lib/client_summarizer.rb

```diff
@@ -0,0 +1,31 @@
+class ClientSummarizer
+  include FiscalYearMixin
+
+  attr_reader :client_namespace, :fiscal_year
```

Comment:
do we need these? since we return a `ClientSummary` instance from `run`, we need an `attr_reader` on `ClientSummary` in order to return `fiscal_year` and `client_namespace`, but I don't see that we are using those attributes on an instance of `ClientSummarizer` directly